include/fw_info.h

```diff
@@ -311,6 +346,16 @@ const struct fw_info_abi *fw_info_abi_get(u32_t id, u32_t index);
 const struct fw_info_abi *fw_info_abi_find(u32_t id, u32_t flags,
 					u32_t min_version, u32_t max_version);
 
+
+/**Invalidate this image by setting the @c valid value to 0x0.
+ *
+ * @note This function needs to have CONFIG_NRF_NVMC enabled.
+ *
+ * @param[in]  fw_info  The info structure to modify.
```

Comment:
Modify is the mechanism, maybe use "invalidate" to be clear about the what the act of modifying this structure specifically is doing?

Reply:
Done